modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/frontend/taglib/clay/data/set/provider/PendingCommerceOrderDataSetDataProvider.java

```diff
@@ -73,8 +73,8 @@
 				pagination.getEndPosition());
 
 		return CommerceOrderClayTableUtil.getOrders(
-			commerceOrders, commerceChannel.getPriceDisplayType(), true,
-			themeDisplay);
+			commerceOrders, null, null, commerceChannel.getPriceDisplayType(),
```

Comment:
why for pending orders you don't pass variable for order type check?

Reply:
the logic has been moved to the CommerceOrderClayTableUtil.java class

Reply:
Yes I saw it, but in order to run that logic you must pass the checker and the other param which in this call are null, I just wonder why?

Reply:
We pass the null value because the new fields must not be displayed on the pending orders page

Reply:
Sorry! You’re right, I’m confused. The order type must be shown on the pending orders page